wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationLegalHoldCell.swift

```diff
@@ -77,32 +77,45 @@ final class ConversationLegalHoldCellDescription: ConversationMessageCellDescrip
     }
 
     private static func configuration(for systemMessageType: ZMSystemMessageType, in conversation: ZMConversation) -> View.Configuration {
-
-        let baseTemplate = "content.system.message_legal_hold"
-        var template = baseTemplate
+        let systemMessageTitle = title(for: systemMessageType)
+        var attributedText = NSAttributedString(string: systemMessageTitle, attributes: ConversationSystemMessageCell.baseAttributes)
 
         if systemMessageType == .legalHoldEnabled {
-            template += ".enabled"
-        } else if systemMessageType == .legalHoldDisabled {
-            template += ".disabled"
+            attributedText = appendLegalHoldEnabledText(to: attributedText)
         }
 
-        var attributedText = NSAttributedString(string: template.localized, attributes: ConversationSystemMessageCell.baseAttributes)
+        let icon = StyleKitIcon.legalholdactive.makeImage(size: .tiny, color: SemanticColors.Icon.foregroundDefaultRed)
 
-        if systemMessageType == .legalHoldEnabled {
-            let learnMore = NSAttributedString(string: (baseTemplate + ".learn_more").localized.uppercased(),
-                                               attributes: [.font: UIFont.mediumSemiboldFont,
-                                                            .link: ConversationLegalHoldSystemMessageCell.legalHoldURL as AnyObject,
-                                                            .foregroundColor: SemanticColors.Label.textDefault])
+        return View.Configuration(attributedText: attributedText, icon: icon, conversation: conversation)
+    }
 
-            attributedText += " " + String.MessageToolbox.middleDot + " "
-            attributedText += learnMore
+    private static func title(for messageType: ZMSystemMessageType) -> String {
+        switch messageType {
+        case .legalHoldEnabled:
+            return L10n.Localizable.Content.System.MessageLegalHold.enabled
+        case .legalHoldDisabled:
+            return L10n.Localizable.Content.System.MessageLegalHold.disabled
+        default:
+            return ""
         }
+    }
 
-        let icon = StyleKitIcon.legalholdactive.makeImage(size: .tiny, color: SemanticColors.Icon.foregroundDefaultRed)
+    private static func appendLegalHoldEnabledText(to attributedText: NSAttributedString) -> NSAttributedString {
+        let learnMoreText = L10n.Localizable.Content.System.MessageLegalHold.learnMore.localizedUppercase
```

Comment:
You can create a localise string as "This conversation is under legal hold [LEARN MORE](%@)"

Reply:
Done! One quick note though: I will keep L10n.Localizable.Content.System.MessageLegalHold.learnMore for now and I will delete it on a separate PR. Because it's being used in one more place and it's a bit more complex to get rid of it there.